requirements/core-requirements.txt

```diff
@@ -0,0 +1,11 @@
+alembic<2.*
+docker<6.*,>=4.0.0
+Flask<3.*
```

Comment:
Is `Flask<3.*` equivalent to `Flask<3`?

Reply:
It is - great point! I've simplified the generate requirements to have the form `<{max_major_version}`